kpi/maintenance_tasks.py

```diff
@@ -0,0 +1,35 @@
+import constance
+from datetime import timedelta
+
+from django.db.models import Exists, OuterRef, Q
+from django.utils import timezone
+
+from kpi.models import AssetSnapshot
+
+
+def remove_old_assetsnapshots():
```

Comment:
Let's be consistent and keep the same name convention. Please rename `remove_old_assetsnapshots` to `remove_old_asset_snapshots`